src/js/modules/articles/reducer.js

```diff
@@ -25,7 +23,17 @@ const reducer = (state = { ...initialState }, action) => {
         ...state,
         isFetching: false,
         isFetched: true,
-        response: action.payload,
+        /* Placing state.articles as the initial value will for some reason
+         * inhibit components' updating with the articles state, which is why
+         * the spread operator is required.
+         */
+        articles: {
+          ...state.articles,
+          ...action.payload.reduce((acc, article) => {
```

Comment:
I'd avoid using a spread + reduce. Very confusing

Reply:
It's really confusing because you're relying on the fact that the spread operator has a lower precedence than reduce. Don't put a spread operator with a reduce. It's not rocket science